### PR TITLE
Modified API to take hours as input and return values accordingly

### PR DESCRIPTION
**Pull request by shubhamjain0594**
This modification is building towards AQI Converter Drivers being written.

### CODE REVIEW

File: smap/sources/conf/twitter_conf.ini [hunk 1]
```diff
@@ -13,4 +13,4 @@ consumer_secret = XXXXXXXXXXXXXXXXXXXXXX
 access_token_key = XXXXXXXXXXXXXXXXXXXXXX
 access_token_secret = XXXXXXXXXXXXXXXXXXXXX
 Timezone = Asia/Kolkata
-url = http://52.20.89.53/
```
Comment (shankari) [inline smap/sources/conf/twitter_conf.ini]: Was this change intentional?

File: smap/sources/safarScraper.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 # used for logging
 import logging
 import sys, traceback
+import simplejson as json
```
Comment (shankari) [inline smap/sources/safarScraper.py]: Do you really need to use simplejson? I thought that python has builtin json support now...

File: air_quality_monitoring/raqmn/raqmn_api/views.py [hunk 3]
```diff
@@ -90,15 +84,17 @@ def get_all_sensors():
...
 	"""
 	Returns the data of sensor specified by uuid of last 30 days
-	TODO : Return the data given time and uuid both, right now only uuid is considered and by default last 30 days data is sent
+	TODO : Return the data given time in days,hours,minutes
 	"""
 	parser = HTMLParser()
 	uuid = parser.unescape(uuid)
 	query_url = 'http://localhost:8079/api/query'
-	query = "select data in (now -30d, now) where uuid='"+uuid+"'"
+	query = "select data in (now -"+hours+"h, now) where uuid='"+uuid+"'"
```
Comment (shankari) [inline air_quality_monitoring/raqmn/raqmn_api/views.py]: So this is the main part of the change, right? Cool that you can change it just by making a small edit to a query.

**Comment by shankari** [comment]
Largely looks good to me. Will pull once you confirm that comment #1 is intentional

File: air_quality_monitoring/raqmn/raqmn_api/views.py (hunk 3 above)
Comment (shubhamjain0594) [inline air_quality_monitoring/raqmn/raqmn_api/views.py]: Yep. This is the only change you have to make in query. All praises for designer for query language for quasar that I suppose is Gabe.

File: smap/sources/conf/twitter_conf.ini (hunk 1 above)
Comment (shubhamjain0594) [inline smap/sources/conf/twitter_conf.ini]: Yes. The twitter driver runs on server, and hence for the server the url is localhost.

File: smap/sources/safarScraper.py (hunk 2 above)
Comment (shubhamjain0594) [inline smap/sources/safarScraper.py]: json has an unicode bug, this was major reason to shift to simplejson.
References: 
[http://stackoverflow.com/questions/712791/what-are-the-differences-between-json-and-simplejson-python-modules](http://stackoverflow.com/questions/712791/what-are-the-differences-between-json-and-simplejson-python-modules)
